universal-application-tool-0.0.1/app/models/Question.java

```diff
@@ -231,4 +235,36 @@ private void setFieldsFromQuestionDefinition(QuestionDefinition questionDefiniti
   public boolean removeVersion(Version draftVersion) {
     return this.versions.remove(draftVersion);
   }
+
+  public boolean addTag(QuestionTag tag) {
+    if (this.questionTags == null) {
+      this.questionTags = new ArrayList<>();
+    }
+    if (this.questionTags.contains(tag)) {
+      return false;
+    }
+    this.questionTags.add(tag);
+    return true;
+  }
+
+  public boolean removeTag(QuestionTag tag) {
+    if (this.questionTags == null) {
+      this.questionTags = new ArrayList<>();
+    }
+    return this.questionTags.remove(tag);
+  }
+
+  public boolean containsTag(QuestionTag tag) {
+    if (this.questionTags == null) {
+      this.questionTags = new ArrayList<>();
+    }
+    return this.questionTags.contains(tag);
+  }
+
+  public List<QuestionTag> getQuestionTags() {
```

Comment:
suggestion (blocking): please document this public method

Reply:
Done